docs/api-reference/next/image.md

```diff
@@ -255,26 +228,135 @@ Other properties on the `<Image />` component will be passed to the underlying
 
 - `style`. Use `className` instead.
 - `srcSet`. Use
-  [Device Sizes](/docs/basic-features/image-optimization.md#device-sizes)
+  [Device Sizes](#device-sizes)
   instead.
 - `ref`. Use [`onLoadingComplete`](#onloadingcomplete) instead.
 - `decoding`. It is always `"async"`.
 
-## Styling
```

Comment:
We might need to add hash redirects to .org for this. And also `Usage` which was removed.